packages/cloud_firestore/test/cloud_firestore_test.dart

```diff
@@ -317,5 +330,26 @@ void main() {
         expect(colRef.path, 'foo/bar/baz');
       });
     });
+
+    group('Query', () {
+      test('getDocuments', () async {
+        final QuerySnapshot snapshot = await collectionReference.getDocuments();
+        final DocumentSnapshot document = snapshot.documents.first;
+        expect(
+          log,
+          equals(
+            <Matcher>[
+              isMethodCall(
+                'Query#getDocuments',
+                arguments: <String, dynamic>{'path': 'foo'},
```

Comment:
Update test to check for expected parameters that would go along with the `getDocuments` call.